.java-version

```diff
@@ -1 +1 @@
-system
+21
```

Comment:
Can this be kept system?
jvm fails if I only have 21.0.1 installed on my system.

Reply:
No sorry, that defeats the whole purpose of using a .java-version file. Don't you use sdkman or jenv? I can have a look with you.

Reply:
(If you mean jenv can't find a matching version for 21, use this to add a symlink for global 21: `ln -s /Users/<username>/Library/Java/JavaVirtualMachines/temurin-21.0.1/Contents/Home ~/.jenv/versions/21`)

Reply:
Adding a symlink is a solution for me, but not for any random person checking out the repo. I guess that's something to figure out as we go.